content/blog/2017/2017-01-18-refactoring-conditional-to-pipeline.adoc

```diff
@@ -0,0 +1,475 @@
+---
+layout: post
+title: "Refactoring Conditional BuildSteps into Jenkins Pipeline"
```

Comment:
Like Tyler mentioned in his earlier comment, I like "migrating" (as you had originally) or "upgrading". :+1: